web/sites/default/config/core.entity_view_display.taxonomy_term.council_district.full.yml

```diff
@@ -105,6 +105,41 @@ third_party_settings:
                 entity: layout_builder.entity
             weight: 1
             additional: {  }
+          9b10e5b6-0e07-4be0-8314-00c7580d6617:
```

Comment:
Was this config intentionally updated?

Reply:
Good catch! I've reverted the change in that file.